tests/unit/test_state.py

```diff
@@ -775,6 +802,138 @@ def test_state_output_unifier_result_converted_to_false(self):
         assert statedecorators.OutputUnifier("unify")(lambda: data)()["result"] is False
 
 
+@skipIf(pytest is None, "PyTest is missing")
```

Comment:
I think that you could just write these as bare functions and get the same effect

Reply:
I just copied the test for states and modified it to verify sub_states.  If both states need this modification I think it should be done separately